x/nft/msg_test.go

```diff
@@ -0,0 +1,94 @@
+package nft
+
+import (
+	"testing"
+
+	"github.com/iov-one/weave/x"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestApprovalMsg(t *testing.T) {
+	Convey("Test add/remove approvals msg validation", t, func() {
+		var helper x.TestHelpers
+		_, validKey := helper.MakeKey()
+		Convey("Test add approvals", func() {
+			Convey("Happy flow", func() {
+				msg := AddApprovalMsg{
+					Id:      []byte("asdf"),
```

Comment:
so `id` is not hex, but all alpha? To me, `id` should be a non collidable, hex or guid/uuid compatible string (can be byte encoded)

Reply:
@alpe I'm not sure why that is, but I guess that has to do with the fact that a username is unique by nature, so we should not be allowing duplicates there anyway